e2e/wdio/headless/test.e2e.ts

```diff
@@ -665,6 +665,30 @@ describe('main suite 1', () => {
                 })
             })
         })
+
+        describe('switchFrame with iframe in shadow DOM', () => {
+            beforeEach(async () => {
+                await browser.url('https://guinea-pig.webdriver.io/iframeInShadowDom.html')
+            })
+
+            it('should switch to iframe inside shadow root via element', async () => {
+                const host = await browser.$('#wrapper')
+                const iframe = await host.shadow$('iframe')
```

Comment:
Why do we fetch this with `shadow$`? Wouldn't a normal user just do
```suggestion
                const iframe = await host.$('iframe')
```